test-app/build.gradle

```diff
@@ -0,0 +1,59 @@
+plugins {
+  id 'net.saliman.properties' version '1.5.2'
+  id "com.marklogic.ml-data-hub" version "6.0.0"
+}
+
+ext {
```

Comment:
I think you can drop this group - now that you have a simple Gradle file for the test-app, the group isn't buying us anything.